inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/convolution.cpp

```diff
@@ -83,9 +83,9 @@ const std::vector<std::vector<ptrdiff_t>> padEnds2D = { {0, 0},
 };
 const std::vector<std::vector<size_t >> dilations2D = { {1, 1},
 };
-const std::vector<size_t> numOutCannels2D = { 1, 2, 5 };
+const std::vector<size_t> numOutCannels2D = { 8, 16, 32};
```

Comment:
Do we have any tests with the wrong parameter which expects exception?

Reply:
Definition of such tests is currently being discussed.
If decided will be added in separate task.